app/schema.json

```diff
@@ -209,14 +212,23 @@
               "children": { "type": "array" }
             }
           },
+          "created": { "type": "string" },
```

Comment:
This shouldn't be here. This is actually erroneously included in exported viewconfs and viewconfs that contain it are incorrect.

Reply:
Would you prefer that the fixture which includes this be moved to view-configs-invalid, or that that fixture just be corrected? (I lean towards the latter.)